README.md

```diff
@@ -112,10 +106,111 @@ SuperGood::SolidusTaxjar.tap do |config|
 end
 ```
 
+### Settings
+
+Developers can configure the following settings in an initializer:
+
+- `cache_duration`: The length of time that cacheable responses from
+  the TaxJar API are cached. Cacheable responses include the list of active
+  nexus regions, as well as tax and tax rate calculations.
+
+  Default value: `3.hours`
+
+- `cache_key`:  function that returns a distinct cache key for
+  a cacheable response from the TaxJar API. A safe default function is provided.
+
+  Default value: (See the source code.)
+
+- `custom_order_params`: A function that returns any parameterized custom
+	order information you need to send to TaxJar in every API request that
+	includes a customer's order. By default no custom order parameters are
+	sent.
+
+	Note that all the essential `Spree::Order`, `Spree::LineItem`, and
+	`Spree::Address` attributes are already sent by default.
+
+	Default value: `{}`
+
+- `discount_calculator`: A discount calculator class. A safe default
+  calculator is provided, but may not be suitable for
+  stores. Check with your accountant how discounts need
```

Comment:
I think the main concern here is that our discount calculator doesn't handle order level adjustments. 

Instead of asking users to talk to their accountants, maybe we should phrase it like this!

```
A discount calculator class. A default
calculator is provided, but may not be suitable for
stores. TaxJar requires discounts to be specified per
line item, so a custom calculator will be required to
support order level discounts and other promotions.
```

Reply:
```
A default calculator is provided, but may not be suitable for stores.
```

I think this statement generalizes a bit and for most stores might not be true. We shouldn't assume that the default calculator is not suitable for most stores, since promotions which rely on order level adjustments are _known_ to not work well with the built-in tax system as well so they are not widely used anymore.

I think as an extension whose main purpose is to provide tax calculations, we should probably phrase this in a way that mentions the exception to the rule, rather than start off with "this might not work for you".

```
A discount calculator class. A default calculator is provided which handles promotions which rely on line item and shipment adjustments. Since TaxJar requires discounts to be specified per line item, you will need to provide a custom version of this calculator in order to support promotions which create order level adjustments. Adjustments to an order made through the Solidus admin would also need to be handled through a custom implementation of this class.
```